src/modules/polls.ts

```diff
@@ -233,28 +229,30 @@ export async function handleCommand(
 			try {
 				const id = interaction.options.getString("id", true);
 				const title = interaction.options.getString("title", true);
-				const cron = interaction.options.getString("cron", true);
 				const channel = interaction.options.getChannel("channel", true);
+				const cron = interaction.options.getString(
+					"schedule",
+					false
+				) as string | null;
 
-				// TODO: don't take this at face value
-				// ^ how important is this? in principle admins won't mess up
```

Comment:
Not all Admins are perfect, unfortunately

Reply:
Will implement checks in upcoming improvements PR